Duplicate identifier in files "*WithoutInternalAllOf*"

When I run yarn build, I encounter errors related to DataSource / DataSourceFromJSON identifiers. 
These files likely need to be added to the exclusions in the script clean_duplicate_import.sh found at https://github.com/flotiq/flotiq-codegen-ts/blob/main/clean_duplicate_import.sh.

```
$ yarn build
yarn run v1.22.21
warning package.json: No license field
$ tsc
src/models/RecipeWithoutInternalAllOfSteps.ts:16:15 - error TS2300: Duplicate identifier 'DataSource'.

16 import type { DataSource } from './DataSource';
                 ~~~~~~~~~~

src/models/RecipeWithoutInternalAllOfSteps.ts:18:5 - error TS2300: Duplicate identifier 'DataSourceFromJSON'.

18     DataSourceFromJSON,
       ~~~~~~~~~~~~~~~~~~

src/models/RecipeWithoutInternalAllOfSteps.ts:23:10 - error TS2300: Duplicate identifier 'DataSource'.

23 import { DataSource, DataSourceFromJSON } from './DataSource';
            ~~~~~~~~~~

src/models/RecipeWithoutInternalAllOfSteps.ts:23:22 - error TS2300: Duplicate identifier 'DataSourceFromJSON'.

23 import { DataSource, DataSourceFromJSON } from './DataSource';
                        ~~~~~~~~~~~~~~~~~~


Found 4 errors in the same file, starting at: src/models/RecipeWithoutInternalAllOfSteps.ts:16

error Command failed with exit code 2.

```

--- a/clean_duplicate_import.sh
+++ b/clean_duplicate_import.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-file_patterns="src/models/*WithoutInternal.ts src/models/*WithoutRequired.ts"
+file_patterns="src/models/*WithoutInternal.ts src/models/*WithoutRequired.ts src/models/*WithoutInternalAllOf*.ts"
 delete_line="import { DataSource, DataSourceFromJSON } from './DataSource';"
 
 escaped_delete_line=$(printf '%s\n' "$delete_line" | sed 's:[][\/.^$*]:\\&:g')

